teams/workstations-canary.yml

```diff
@@ -36,12 +37,11 @@ queries:
   interval: 300
   observer_can_run: true
   automations_enabled: false
-policies: 
-- path: ../lib/macos-device-health.policies.yml
-- path: ../lib/windows-device-health.policies.yml
-- path: ../lib/windows-device-health.policies.yml
-agent_options: ./lib/agent-options.yml
+policies:
+  - path: ../lib/macos-device-health.policies.yml
+  - path: ../lib/windows-device-health.policies.yml
```

Comment:
Similar comment about indentation here. Just want to make sure we're being consistent w/ other options that accept arrays (`custom_settings`, `scripts`, etc.)